send_email_messages/frontend/settings.js

```diff
@@ -0,0 +1,255 @@
+import { 
```

Comment:
Generally curious how this approach compares vs. the prior version which stored config in airtable. Does this provide more flexibility? How easy is it to copy settings across cities?

Reply:
I think this approach is an improvement because:
* it is (or could be, if it's not already) self-documenting. Plenty of room in the UI to explain what each of these things does and what a normal value is
* validatable. We can make sure they pick a column that actually exists, or for "stage" they pick a stage that exists on that column.
* robust to column name changes - any time we're representing a field in globalConfig we represent it by id, which remains the same even if the field name changes. But the settings UI will update to reflect the new name
* harder to mess up. It seems more likely that your typical curious city administrator will go poking around an airtable table, maybe inadvertently add an empty row which could mess things up. This seems clearly designed for "I know what I'm doing"

However, it does have disadvantages:
- can't copy&paste from city to city -- each new city we deploy to, someone will have to go through and set up the email types, template vars, etc. using the settings pane in the emails block (however, this isn't actually that much of a regression. When deploying to Walnut Creek, I was able to copy the config in from paterson, but I had to change almost every field because the column names were somewhat different. I also had to re-set-up the column names and field types in "Email Configuration" (they didn't copy over) and had to make sure it was just so or the code would break). so ... there was already a lot of manual work of copying over config, and I think this way makes it actually smoother to copy it over and more likely that it'll be copied accurately